manim/animation/creation.py

```diff
@@ -1,3 +1,4 @@
+"""Display a mobject by fading it in, or stop displaying it by fading it out."""
```

Comment:
```suggestion
"""Animate the addition or removal of a mobject from a Scene."""
```
Nit: This sounds too specific, as if it only applies to `FadeIn` and `FadeOut`.

Reply:
lol I didn't catch that thanks

Reply:
I changed it to  `Animate the display or removal of a mobject from a scene`, as we are trying to use the verb "display" for adding a mobject to the scene